index.html

```diff
@@ -235,50 +235,50 @@ <h1 class="siteAboutContainer animated section-header">
+                            <h2>No upcoming events</h2>
                         </div>
                     </div>
                     <div class="row eventInfo">
                         <div class="col-xs-7 eventDesc">
                             <div class="row eventDescInfo">
                                 <div class="col-lg-12">
                                     <p class="eventDescInfoContainer">
-                                        <p>An introduction to the world of programming, CS careers and more</p>
+                                        <p>No upcoming event for now !!</p>
                                     </p>
                                 </div>
                             </div>
                         </div>
                         <div class="col-xs-5 eventTiming text-left">
                             <div class="date">
                                 <i class="fa fa-calendar icon"></i>&nbsp;
-                                <span><strong>8 Aug '19</strong></span>
+                                <span><strong>-</strong></span>
                             </div>
                             <div class="startTimimg">
                                 <i class="fa fa-clock-o icon"></i>&nbsp;
-                                <span><strong>2:00 PM</strong></span>
+                                <span><strong>-</strong></span>
                             </div>
                             <div class="endTimimg">
                                 <i class="fa fa-clock-o icon"></i>&nbsp;
-                                <span><strong>4:00 PM</strong></span>
+                                <span><strong>-</strong></span>
                             </div>
                             <div class="eventLocation">
                                 <i class="fa fa-map-marker icon"></i>&nbsp;
-                                <span><strong>E3-104</strong></span>
+                                <span><strong>-</strong></span>
                             </div>
                             <div class="eventOD">
-                                <i class="fa fa-thumbs-up icon text-success"></i>&nbsp;
-                                <span><strong>OD</strong></span>
+                                <i class="fa fa-thumbs-down icon text-danger"></i>&nbsp;
+                                <span><strong>No OD</strong></span>
                             </div>
                         </div>
                     </div>
                     <div class="row eventCTA">
                         <div class="col-xs-12 text-center">
-                            <a href="https://asetalias.in" target="_blank">
-                                <button class="btn btn-danger btn-lg btn-flat">Open for all freshers, no registration required</button>
+                            <a href="https://docs.google.com/forms/d/e/1FAIpQLScpSM7nisQQKlBB5oYdkeo2cMV6I0b__F_KowLgbUJePyyLZg/viewform?usp=sf_link" target="_blank">
```

Comment:
@TanyaaCJain @thisisayush  I think we deprecated this form in favor of using the logistics repo as event issues and queries.